roles/foreman_provisioning/tasks/configure_centos_9.yml

```diff
@@ -7,15 +13,16 @@
       - x86_64
     media:
       - CentOS Stream 9 mirror
-    provisioning_templates:
-      - Kickstart default
+    provisioning_templates: "{{ result.resources | map(attribute='name') }}"
     ptables:
       - Kickstart default
     state: present
+    password_hash: "SHA256"
```

Comment:
We made this the default, but I don't mind making it explicit now because we didn't add a migration to change existing OSes.

Reply:
Ah yes, probably Adam didn't have https://github.com/theforeman/foreman-ansible-modules/pull/1779 in his setup :)

Reply:
I indeed did not have that

Reply:
Heh, I recall fixing that after Archana run into the same thing when initially writing this play.

Reply:
Funny, I thought it was related to https://github.com/theforeman/foreman/commit/e2dee7d6f7ffb505d317ca1b205f302ccd622063 but now that I read the PR itself it should really be the more secure SHA512.
```suggestion
    password_hash: "SHA512"
```

Reply:
https://github.com/theforeman/forklift/pull/1886